lib/suspenders.rb

```diff
@@ -1,4 +1,7 @@
 require 'suspenders/version'
 require 'suspenders/generators/app_generator'
 require 'suspenders/actions'
+require 'suspenders/adapters/heroku'
 require 'suspenders/app_builder'
+
+ROOT_PATH = File.expand_path '../..', __FILE__
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.